installkernel

```diff
@@ -78,9 +78,16 @@ updatever() {
 # into kernel package installation.  Also make sure the PATH includes
 # /usr/sbin and /sbin, just as dpkg would.
 if [ "${dir}" = "/boot" ]; then
-	PATH="${PATH}:/usr/sbin:/sbin" \
-		run-parts --verbose --exit-on-error --arg="${ver}" --arg="${img}" \
-		/etc/kernel/preinst.d
+	(
+		export LC_ALL=C PATH="${PATH}:/usr/sbin:/sbin"
```

Comment:
why do we need to change PATH?

Reply:
It's just preserving existing behaviour, I think.

Reply:
Because the previous code did so I preserved it.